### ISSUE TITLE
suport to configure exporting metric point in the minute that not include seconds

### ISSUE THREAD

**Issue by JianhuaGong** [open-telemetry/opentelemetry-java#4171]
**Is your feature request related to a problem? Please describe.**
Now, we use version 1.6.0 to record metrics and export metrics per minute, the report time of metric point often includes seconds like 12:01:25, 12:02:25, then the time becomes 12:10:36, 12:11:36 after application restarting.

**Describe the solution you'd like**
we want to configure exporting metric point in the minute that not include seconds.

**Describe alternatives you've considered**
PeriodicMetricReaderFactory add a parameter 'delay' and PeriodicMetricReader use it in scheduler.scheduleAtFixedRate

**Additional context**
none

**Comment by JianhuaGong** [comment]
It is not a problem, it is improved features.  Metric in the minute that not include seconds is consistent with metric of other system, and looks a little more professional.

**Comment by jsuereth** [comment]
TL;DR: I think (perhaps), using a time-alignment function (e.g. rate/increase) in your metric query *may* give you better results rather than truncating/altering the data on ingestion.

With that, here's my two thoughts on this request:

1. Adding a delay before scheduleAtFixedRate is totally something that we can add to PeriodicMetricReader.   The code this is based on (IntervalMetricReader in OpenCensus, almost direct-ported to OTEL at one point) allowed a delay before scheduling, as "time to first metric" post-startup can be an important tuning parameter.   We could use this to do our best to line up reporting intervals w/ round minutes, but it's somewhat impossible to guarantee.
2. Given that `MetricExporter` is a public interface, you should be able to create your own wrapper around a MetricExporter that truncates the timestamp to the closest minute.   Note: Depending on your backend, this could actually cause (wild) inaccuracies in e.g. RATE calculations on Counters, but would accomplish what you want.   I think it'd be good for you to test out such a feature with a custom extension on MetricExporter before we'd add it to the OTEL SDK.   Specifically, I'm worried about the oddities it could cause query-side and would like to understand better your use case and whether you run into this.   Particularly, depending on your backend and instrument type, this may not be an option.

If you can describe your metrics better, we can see if we can find the best solution for you.

**Comment by jack-berg** [comment]
Couldn't you accomplish this by configuring `SdkMeterProviderBuilder` with a custom clock implementation that truncated the `now()` to the desired granularity?
